CHANGES.txt

```diff
@@ -4,6 +4,7 @@ Changelog for Shake (* = breaking change)
     #689, allow adding help messages at the end using addHelpSuffix
     #686, rename Build.hs to Shakefile.hs
     #690, make Ninja mode use -j0 by default (more like Ninja)
+    #706, rename Verbosity's constructors and add Warn
```

Comment:
I usually put these things at the top of the changelog, FWIW - however, it doesn't really matter and maybe this is a better way to avoid conflicts. However, growing up makes it easier to see the diff between prereleases.